pimd/pim6_cmd.c

```diff
@@ -451,6 +451,33 @@ DEFPY (no_ipv6_pim_rp_prefix_list,
 	return pim_process_no_rp_plist_cmd(vty, rp_str, plist);
 }
 
+
+DEFPY (ipv6_ssmpingd,
+      ipv6_ssmpingd_cmd,
+      "ipv6 ssmpingd [X:X::X:X]$source",
+      IPV6_STR
+      CONF_SSMPINGD_STR
+      "Source address\n")
+{
+	const char *src_str = (source_str) ? source_str : "::";
```

Comment:
nit: Redundant `()`

Reply:
i am not sure if its redundant. We could have it as it is.